the-wing/components
# PAMS-1950 video player added

## Description
Added Video Player Component
## Jira Ticket


## Screenshots
![image](https://user-images.githubusercontent.com/22821635/51038538-2c226f80-15bc-11e9-9843-1544fd4e0008.png)

## How should this be manually tested?
1. In UI, go to the video section
2. click default -> Video Component should mount, not autoplay, and not be muted once playing
3. click autoplay -> Video Component should mount, autoplay, and not be muted once playing
4. click muted -> Video Component should mount, not autoplay, and be muted once playing

## Type of change


## Checklist:
